activemodel/lib/active_model/validations/length.rb

```diff
@@ -64,8 +64,8 @@ def validate_each(record, attribute, value)
       end
 
       private
-        def skip_nil_check?(key)
-          key == :maximum && options[:allow_nil].nil? && options[:allow_blank].nil?
+        def skip_nil_check?(key, value)
+          (key == :maximum && options[:allow_nil].nil? && options[:allow_blank].nil?) || (key == :minimum && value.nil?)
```

Comment:
@rafaelfranca This test case https://github.com/rails/rails/blob/0f87fce024f87a0d691a235f351de6c5be2026bb/activemodel/test/cases/validations/length_validation_test.rb#L391 is contradictory against ours. Will i update it also?

Reply:
updated it accordingly